libraries/messages/src/logging/macros.rs

```diff
@@ -0,0 +1,50 @@
+macro_rules! display_event {
+    ($( [$e:ident, $s:literal$(,)? $( $p:ident ),* ] ),*) => {
```

Comment:
😵‍💫 I am confused. Not very self documenting, maybe a brief comment of what is going on.